apis/core/hydra/projects.ttl

```diff
@@ -9,7 +9,24 @@ cc:CubeProject
   a
     hydra:Class ;
   query:include
-    cc:csvMapping .
+    cc:csvMapping ;
+  hydra:supportedOperation
+  [
+    a 
+      hydra:Operation, 
+      cc:DeleteProjectAction ;
```

Comment:
I would replace `cc:DeleteProjectAction` with `schema:DeleteAction`

Reply:
Should I also remove cc:DeleteCSVSourceAction to be consistent?

Reply:
I think I handled it in https://github.com/zazuko/cube-creator/pull/108